src/luarocks/cmd.lua

```diff
@@ -360,6 +360,9 @@ local function get_config_text(cfg)  -- luacheck: ignore 431
    local libdir_ok = deps.check_lua_libdir(cfg.variables)
    local incdir_ok = deps.check_lua_incdir(cfg.variables)
    local lua_ok = cfg.variables.LUA and fs.exists(cfg.variables.LUA)
+   if hardcoded.FORCE_HARDCODED then
+      lua_ok = hardcoded.LUA
```

Comment:
Why is it not in the `cfg` at this stage? I see that cfg is an input parameter. Does not it contain correct lua interpreter already?

Reply:
Now here is cfg.variables.LUA